notifications_utils/formatters.py

```diff
@@ -287,6 +288,83 @@ def normalise_whitespace(value):
     return ' '.join(strip_and_remove_obscure_whitespace(value).split())
 
 
+def get_action_links(html: str) -> list[str]:
+    """Get the action links from the html email body and return them as a list. (insert_action_link helper)"""
+    # set regex to find action link in html, should look like this:
+    # &gt;&gt;<a ...>link_text</a>
+    action_link_regex = re.compile(
+        r'(>|(&gt;)){2}(<a style=".+?" href=".+?"( title=".+?")? target="_blank">)(.*?</a>)'
+    )
+
+    return re.findall(action_link_regex, html)
+
+
+def get_img_link() -> str:
+    """Get action link image url for the current environment. (insert_action_link helper)"""
+    img_link = 'https://dev-va-gov-assets.s3-us-gov-west-1.amazonaws.com/img/vanotify-action-link.png'
+
+    if os.environ.get('NOTIFY_ENVIRONMENT') == 'production':
+        img_link = 'https://prod-va-gov-assets.s3-us-gov-west-1.amazonaws.com/img/vanotify-action-link.png'
+    elif os.environ.get('NOTIFY_ENVIRONMENT') in ['staging', 'performance']:
+        img_link = 'https://staging-va-gov-assets.s3-us-gov-west-1.amazonaws.com/img/vanotify-action-link.png'
+
+    return img_link
+
+
+def insert_action_link(value: str) -> str:
```

Comment:
Tinny tiny nit: I feel the arg 'value' should 'html' like in `get_action_links `method